src/pl/senla/hotel/annotations/config/ConfigPropertyAnnotationLoader.java

```diff
@@ -0,0 +1,78 @@
+package pl.senla.hotel.annotations.config;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Properties;
+import java.util.Set;
+
+public class ConfigPropertyAnnotationLoader {
+
+    private static final String EMPTY = "";
+    private final String configDirectory;
+    private final String configName;
+
+    public ConfigPropertyAnnotationLoader(String configDirectory, String configName) {
+        this.configName = configName;
+        this.configDirectory = configDirectory;
+    }
+
+    public void load(Set<Field> fields) throws IllegalAccessException, InvocationTargetException, InstantiationException {
```

Comment:
annotation loader better to integrate into your DI loader to inject not only necessary dependencies but also required config props

Reply:
ok